kiwi/tasks/system_build.py

```diff
@@ -155,11 +155,12 @@ def process(self):
         system = SystemPrepare(
             self.xml_state, image_root, True
         )
+
         manager = system.setup_repositories()
-        system.install_bootstrap(manager)
-        system.install_system(
-            manager
-        )
+        if self.xml_state.get_bootstrap_packages_sections():
+            system.install_bootstrap(manager)
+        system.install_system(manager)
+
```

Comment:
My initial intention was to let bootstrap run normally as we have been doing up to now (if not boostrap section it shouldn't hurt, just verify package manager is there and not much more), however it turned out that the debian boostraping tool is not really re-entrant and I didn't want to code any strange check to guess if the bootstrap was already there or not. That's why I came with the idea of not even running bootstrap if not stated in the description file.

I think all three cases you noted above are covered with the condition I wrote, since `xml_state.get_bootstrap_packages_sections` returns any `<packages type="bootstrap">`.

Said that I am totally opened to modify that condition or find another solution.